### ISSUE TITLE
Accessibility button does not appear (at least) on Safari 15 on MacOS 12 Monterey

### ISSUE THREAD

**Issue by woluweb** [joomla/joomla-cms#36311]
### Steps to reproduce the issue
This refers to https://www.facebook.com/groups/joomlanospam/posts/10158088758700997/
As can be seen on https://www.whatt.eu/en/ the accessibility button (bottom left, just above the cookie button, does appear on all browsers except (at least) on Safari 15 on MacOS 12 Monterey

See the Facebook thread for more screenshots and feedbacks

**Comment by TLWebdesign** [comment]
The accessibility javascript tries to load the material icons. On safari for some reason it can't and then loads an alternative which has the transparent background on an emoticon instead of the default material icon. 

This is the console message: 
[Warning] Accessibility: Material Icons font was not loaded, using emojis instead (accessibility.min.js, line 1:2860)

Seem JS can't add the stylesheet to the page and that check fails so it adds a backup that is not styled correctly.

Kind regards,
Tom

**Comment by infograf768** [comment]
OS Monterey

Firefox
<img width="451" alt="Screenshot 2021-12-15 at 07 31 13" src="https://user-images.githubusercontent.com/869724/146135467-e8495e3a-2ae7-4cb7-81df-df81dedfbfa9.png">

Safari 15

<img width="451" alt="Screenshot 2021-12-15 at 07 31 47" src="https://user-images.githubusercontent.com/869724/146135508-2fcfff31-cbea-4728-b887-2f0ba318a74a.png">

Therefore missing not only the font family (Material Icons) but also background colour and the font is set to Italic

**Comment by brianteeman** [comment]
It's all the same issue. Will pr later today

**Comment by brianteeman** [comment]
https://github.com/ranbuch/accessibility/issues/24

**Comment by infograf768** [comment]
Simplest way to solve the issue in the js is correcting the error:

It should be `fontFamily:"Material+Icons",` and not `fontFamily:"Material Icons",`

**Comment by infograf768** [comment]
https://github.com/ranbuch/accessibility/pull/27

**Comment by brianteeman** [comment]
I think you are confused between fontFamily and fontFaceSrc

**Comment by infograf768** [comment]
> I think you are confused between fontFamily and fontFaceSrc

Nope. The js contains both loading the font family and a redundant font-family for the css.
Deleting that second one also corrects the issue as the family is already defined. I just added the `+` to simplify.

**Comment by angieradtke** [comment]
Hi Guys,
 It take a closer look at this plugin and its code today.  The idea behind is great, but the HTML-output should be changed.
The reason is the accessibility itself:

-  `<i>`  for the icons should be changed  to native  buttons.
- The google icon font should be replaced by svg-icons,
- The reset button has to become more clear for better understanding
- Close und reset button should be semantically at the and of the wrapping` <div>`

Maybe it is something for @dgrammatiko 
I can do the icons

**Comment by brianteeman** [comment]
All of that must be made upstream

**Comment by angieradtke** [comment]
what do you mean - at ranbuch account?

**Comment by angieradtke** [comment]
https://github.com/angieradtke/Accessibility-plugin-icons/blob/main/README.md

**Comment by bembelimen** [comment]
> The google icon font should be replaced by svg-icons,

The important thing is to get rid of Google fonts at all, as we have a GDPR problem here in Europe with Joomla! otherwise (or opt-in at least)

**Comment by brianteeman** [comment]
@angieradtke these changes can only be made upstream at ranbuch.

**Comment by angieradtke** [comment]
Benjamin told be we forked  it already

**Comment by bembelimen** [comment]
> Benjamin told be we forked  it already

Sorry I was wrong, I thought about another script there

**Comment by angieradtke** [comment]
I think needed changes are so massiv that a fork could make sence @dgrammatiko  @chmst

**Comment by ghost** [comment]
> > The google icon font should be replaced by svg-icons,
> 
> The important thing is to get rid of Google fonts at all, as we have a GDPR problem here in Europe with Joomla! otherwise (or opt-in at least)

I don't know who downvoted this comment, but for german users without an annoying cookie-consent, using this plugin is not possible.  
With the new law ttdsg, the users needs to consent before any request to google or other non european country is allowed.  
The use of svg icons or a minimal font-icon-set from icomoon local hosted, would be the perfect solution.  

https://www.gesetze-im-internet.de/ttdsg/  
https://rewis.io/urteile/urteil/2tj-01-12-2021-6-l-73821wi/  
https://rewis.io/urteile/urteil/lhm-20-01-2022-3-o-1749320/

**Comment by brianteeman** [comment]
Among other things this is resolved by #36948

**Comment by woluweb** [comment]
I have migrated 6 sites from 4.0 to 4.1.
They now all have the following error in the Console:

```
Uncaught (in promise) DOMException: Failed to execute 'check' on 'FontFaceSet': Could not resolve '1em Material+Icons' as a font.
    at i (https://www.___.be/media/vendor/accessibility/js/accessibility.min.js?3.0.15:1:2738)
    at https://www.___.be/media/vendor/accessibility/js/accessibility.min.js?3.0.15:1:2786
```

The Accessibility button looks fine (with the wheelchair icon), but when opening it all icons there are broken.

NB: in the meatime I see this issue had just been reported (with a fix mentioned by @brianteeman) : https://github.com/joomla/joomla-cms/issues/37050

**Comment by brianteeman** [comment]
Its because the change made in the script was wrong and merged even though I stated several times it was wrong :(

**Comment by brianteeman** [comment]
see #38009

**Comment by alikon** [comment]
please test #38009
